src/main/kotlin/agallipo/BoardingCardSorter.kt

```diff
@@ -1,114 +1,114 @@
-package agallipo
+    package agallipo
 
-import java.io.ObjectInputFilter.merge
+    import java.io.ObjectInputFilter.merge
+    //created by alex :)
+    interface BoardingCardSorter {
```

Comment:
Why interface if you implement inside all the methods?

Reply:
It´s ok to create an interface but it is not usual to implement all the methods inside the interface, usually in another class. The motive to create an interface, i think, is that if tomorrow the data is provided in a text file, or in xml, or anything else, you only have to create the part o reading these new input, and the rest of the code will be the same.

Reply:
I just made it a class, maybe now makes more sense, don't you agree?